Framework/qcTaskDplConfig.ini

```diff
@@ -0,0 +1,94 @@
+;===============================
+; General
+;-------------------------------
+
+[DataSampling]
+# list of tasks that need Data Sampling
+tasksList=skeletonTask
+# Generate time-pipelined dispatchers if data sources are time-pipelined. When disabled, one dispatcher will be generated for the same time-pipelined Data Processors.
+enableTimePipeliningDispatchers=0
+# Generate parallel dispatcher for each parallel flow of data (having different subSpecs). When disabled, one dispatcher will be generated for all parallel flows.
+enableParallelDispatchers=0
+# Now it does nothing.
+enableProxy=0
+
+[database]
+;implementation=MySql
+username=qc_user
+password=qc_user
+;host=localhost
+name=quality_control
+implementation=CCDB
+host=localhost:8080
+
+[Activity] # Similar to a "run"
+number=42
+type=2
+
+;===============================
+; Tasks
+;-------------------------------
+[skeletonTask]
+taskDefinition=skeletonTaskDefinition
+
+[skeletonTaskDefinition]
+className=o2::quality_control_modules::skeleton_dpl::SkeletonTaskDPL
+moduleName=QcSkeletonDpl
+moduleOfChecks=QcExample
+maxNumberCycles=-1
+outputDataOrigin=ITS
+outputDataDescription=HIST_SKLT_TASK
+#inputs list
+inputs=readoutInput
+# Fraction of data that should be bypassed to QC task. The less, the better performance.
+fraction=0.1
+
+[readoutInput]
+inputName=readout
+dataOrigin=ITS
+dataDescription=RAWDATA
+
+;===============================
+; Checkers
+;-------------------------------
+
+[checkers] ; needed for the time being because we don't have an information service
+numberCheckers=1
+numberTasks=1
+tasksAddresses=tcp://localhost:5556,tcp://localhost:5557,tcp://localhost:5558,tcp://localhost:5559
+
+[checker_0]
+broadcast=1
+broadcastAddress=tcp://*:5600
+id=0
+
+[Checks]
+checkMeanIsAbove/threshold=1
+
+;===============================
+; Monitoring
```

Comment:
I think this section is not needed anymore as currently Monitoring library creates an instance based on the provided URL: see https://github.com/AliceO2Group/QualityControl/blob/40ac89f825cce72a3d8109e01b681889da61c570/Framework/src/TaskDevice.cxx#L60

Reply:
So can I delete it also here? https://github.com/AliceO2Group/QualityControl/blob/40ac89f825cce72a3d8109e01b681889da61c570/Framework/example-default.ini#L83
And, since both refer to this [file](https://github.com/AliceO2Group/QualityControl/blob/master/Framework/apmon.cfg) , can it be dropped it as well?